Source/Core/Core/CoreParameter.cpp

```diff
@@ -402,28 +402,42 @@ void SCoreStartupParameter::CheckMemcardPath(std::string& memcardPath, std::stri
 	}
 }
 
-IniFile SCoreStartupParameter::LoadGameIni() const
+IniFile SCoreStartupParameter::LoadDefaultGameIni() const
 {
 	IniFile game_ini;
-	game_ini.Load(m_strGameIniDefault);
-	if (m_strGameIniDefaultRevisionSpecific != "")
-		game_ini.Load(m_strGameIniDefaultRevisionSpecific, true);
-	game_ini.Load(m_strGameIniLocal, true);
+	LoadGameIni(&game_ini, File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP);
 	return game_ini;
 }
 
-IniFile SCoreStartupParameter::LoadDefaultGameIni() const
+IniFile SCoreStartupParameter::LoadLocalGameIni() const
 {
 	IniFile game_ini;
-	game_ini.Load(m_strGameIniDefault);
-	if (m_strGameIniDefaultRevisionSpecific != "")
-		game_ini.Load(m_strGameIniDefaultRevisionSpecific, true);
+	LoadGameIni(&game_ini, File::GetUserPath(D_GAMESETTINGS_IDX));
 	return game_ini;
 }
 
-IniFile SCoreStartupParameter::LoadLocalGameIni() const
+IniFile SCoreStartupParameter::LoadGameIni() const
 {
 	IniFile game_ini;
-	game_ini.Load(m_strGameIniLocal);
+	LoadGameIni(&game_ini, File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP);
+	LoadGameIni(&game_ini, File::GetUserPath(D_GAMESETTINGS_IDX));
 	return game_ini;
 }
+
+void SCoreStartupParameter::LoadGameIni(IniFile* game_ini, const std::string& path) const
+{
+	// INIs with region wildcards
+	if (GetUniqueID().size() >= 4)
+	{
+		std::string id_without_region(GetUniqueID());
+		id_without_region.replace(3, 1, "-");
+		game_ini->Load(File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP + id_without_region + ".ini", true);
```

Comment:
"game_ini->Load(File::GetSysDirectory() + GAMESETTINGS_DIR DIR_SEP" should be "path"